ui/src/main/js/application/auth/AuthenticationConfiguration.js

```diff
@@ -3,7 +3,7 @@ import CommonGlobalConfigurationForm from 'common/configuration/global/CommonGlo
 import * as FieldModelUtilities from 'common/util/fieldModelUtilities';
 import { CONTEXT_TYPE } from 'common/util/descriptorUtilities';
 import * as PropTypes from 'prop-types';
-import CommonGlobalConfiguration from 'common/configuration/global/CommonGlobalConfiguration';
+import PageHeader from '../../common/components/navigation/PageHeader';
```

Comment:
We purposely changed our webpack file so that we can reference the directories of the project to have what appears to be absolute paths from the project root rather than relative paths.  We wanted the import paths to behave like imports of packages in Java. 
```suggestion
import PageHeader from 'common/components/navigation/PageHeader';
```

Reply:
I'll look into this one shortly.  I might have had to rebuild for it to recognize the path, but when developing this yesterday, the relative path seemed to be working.

Reply:
The relative paths should work to it's just a style we try.  We want to use a full path from the source root directory if possible so that there's no ambiguity where the component is being imported from.

Reply:
Fixed this in latest commit 👍